### PR TITLE
Update usage.mdx

### PR DESCRIPTION
**Pull request by mbgower**
Suggested text and images updates for Progress bar. Addresses some of the issues identified in the [accessibility review ](https://ibm.box.com/s/dnj72eer3x8a6ppdx9694zy88tbncr5k)of the component.

### CODE REVIEW

**Review by tay1orjones — APPROVED** [review]

**Comment by mbgower** [comment]
BTW, @aagonzales, there's a Loading: skeleton states link at the bottom that is a 404. I couldn't figure out what it was supposed to be pointing at. I also wasn't sure why it was broken out in a second Components list.

**Comment by aagonzales** [comment]
Adding @thyhmdo as a reviewer since she is closest to this information.

File: src/pages/components/progress-bar/usage.mdx [hunk 1]
```diff
@@ -368,16 +368,17 @@ dismissed depending on what is most suitable for the use case.
 
 The active loading state indicates that the action is still in progress.
 
-#### Success
+<Row>
+<Column colLg={12}>
 
-The success loading state indicates that the action completed successfully.
+![Progress bar as a part of a page](images/progress-bar-usage-19.png)
```
Comment (aagonzales) [inline src/pages/components/progress-bar/usage.mdx:374]: I'm not sure moving this image up works here. It's very big for the context. If we want an example after each state, I'd suggest, smaller cropped versions. And perhaps a table somewhere so you can see all the definitions in one view without scrolling so much

File: src/pages/components/progress-bar/usage.mdx [hunk 2]
```diff
@@ -95,8 +95,8 @@ which a bar is moving along in constant speed, repeating over time.
...
-| [Determinate](Determinate)                                                 | Use determinate progress bars for a specified amount of time for clear information about the progression — they convey exact and qualitative information about the progression.                                                                            |     |
-| [Indeterminate](/components/progress-bar/usage#indeterminate-progress-bar) | Use indeterminate progress bars for an unspecified completion of time or unclear information about the progression — they convey that users’ request, action, or data is being processed at the moment without indicating how long the activity will take. |
+| [Determinate](Determinate)                                                 | Use determinate progress bars where there is clear information about the progression — they convey exact and qualitative information about the progression.                                                                            |     |
```
Comment (aagonzales) [inline src/pages/components/progress-bar/usage.mdx:98]: Isn't "specified" and "unspecified" amount of time an important differentiator  to mention? Why did you think its not necessary, just curious

Comment (aagonzales) [reply src/pages/components/progress-bar/usage.mdx:98]: Also I'll defer to @thyhmdo on this change.

**Comment by aagonzales** [comment]
This is probably the correct link to loading skeleton states: https://carbondesignsystem.com/patterns/loading-pattern/#skeleton-states

File: src/pages/components/progress-bar/usage.mdx (hunk 2 above)
Comment (mbgower) [reply src/pages/components/progress-bar/usage.mdx:98]: I really didn't find those terms meaningful. Few author-based progress bars measure a specific _time_, right? It's typically a specific amount of data or percentage of progression. Except in major browser-level downloads, it's rare to see time mentioned (and tends to be woefully inaccurate without quite a bit of data to base that on).

Comment (thyhmdo) [reply src/pages/components/progress-bar/usage.mdx]: I think it makes sense and you're right. Time is usually hard to define and that sentence was too specific about timing while there are other ways to provide this type of information.

**Review by aledavila — APPROVED** [review]
lgtm

**Comment by mbgower** [comment]
This PR has been displaced by https://github.com/carbon-design-system/carbon-website/pull/2978

**Pull request closed without merging**